### PR TITLE
04 - add new columns to oauth_apps

### PR DESCRIPTION
**Pull request by alrocar**
Related to #15015

Depends on:

- https://github.com/CartoDB/cartodb/pull/15024
- https://github.com/CartoDB/cartodb-central/pull/2538

### CODE REVIEW

File: app/controllers/carto/controller_helper.rb [hunk 1]
```diff
@@ -64,11 +64,11 @@ def rescue_from_record_not_found
       render_jsonp({ errors: 'Record not found' }, 404)
     end
 
-    def rescue_from_central_error
-      CartoDB::Logger.error(exception: e,
+    def rescue_from_central_error(error)
```
Comment (ethervoid) [inline app/controllers/carto/controller_helper.rb]: :heart:

**Comment by gonzaloriestra** [comment]
:heavy_check_mark: Acceptance OK

**Comment by gonzaloriestra** [comment]
retest this please

**Comment by gonzaloriestra** [comment]
retest this please

**Comment by gonzaloriestra** [comment]
retest this please

**Comment by gonzaloriestra** [comment]
retest this please

**Comment by gonzaloriestra** [comment]
retest this please

**Pull request merged**